dist/docker/debian/build_docker.sh

```diff
@@ -76,7 +76,7 @@ run apt-get -y update
 run apt-get -y install dialog apt-utils
 run bash -ec "echo 'debconf debconf/frontend select Noninteractive' | debconf-set-selections"
 run bash -ec "rm -rf /etc/rsyslog.conf"
-run apt-get -y install hostname supervisor openssh-server openssh-client openjdk-11-jre-headless python python-yaml curl rsyslog locales sudo
+run apt-get -y install hostname supervisor openssh-server openssh-client openjdk-11-jre-headless python2.7 python3-yaml curl rsyslog locales sudo
```

Comment:
Adding python3-yaml made it even stranger that you don't list python3 on the requirement list and assume that something else already installed it. Why not add python3 here too, even if it's not really necessary?

Reply:
@nyh I can add it as well.. although i think maybe we should just add it to `install-dependencies.sh , WDYT?

Reply:
I'm not sure I follow all the details here. install-dependencies.sh is for **building** Scylla, not for running it. This script, that's for **running** scylla, no? If it's for running Scylla, and python3 is needed for running, then I think this script should mention it - even if it's needed for building as well.

Reply:
In any case, I'm not saying there's anything _wrong_ with your patch - I'm pretty sure that "python3-yaml" depends on "python3" so you'll get python3 even if you don't mention it explicitly. I just thought it's cleaner to mention it explicitly.

Reply:
Ok, added python3 as well so it will more clear